build-release-tools/application/release_to_atlas.py

```diff
@@ -9,20 +9,18 @@
     on-tools/manifest-build-tools/application/release_to_atlas.py \
     --build-directory b/RackHD/packer \
     --atlas-url https://atlas.hashicorp.com/api/v1 \
-    --atlas-username rackhd \
+    --atlas-creds rackhd:****** \
     --atlas-name rackhd \
-    --atlas-token ****** \
     --atlas-version 1.2.3 \
     --is-release true
 
 The required parameters:
     build-directory: A directory where box files laid in.
-    atlas-token: atlas access token.
+    atlas-creds: value or env var name of rackhd:****.
```

Comment:
Can you please comment in code for the syntax ?
say, USER:API-TOKEN ( or password ? )
 then it's a 👍